### PR TITLE
docs: add a hint to release notes in landing page

### PR DESCRIPTION
**Pull request by sarahgm**
# Description

Add a hint at the landing page for the release blog
# Reviewers:
@marigold-ui/developer
@marigold-ui/designer

### CODE REVIEW

File: docs/ui/ShinyText.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  return (
+    <p
+      style={
+        {
+          '--shiny-width': `${shinyWidth}px`,
+        } as CSSProperties
+      }
+      className={cn(
+        'mx-auto max-w-md text-neutral-600/70',
+
```
Comment (aromko) [inline docs/ui/ShinyText.tsx:26]: I don't think the blankline and comments are necessary?

**Review by aromko — DISMISSED** [review]

File: docs/app/page.tsx [hunk 2]
```diff
@@ -32,191 +34,210 @@ const SectionLink = ({
...
+  return (
+    <main className="grid justify-center px-[--page-padding-md] xl:px-[--page-padding-xl]">
+      {/* Hero */}
+      <div className="grid h-[calc(100dvh-var(--page-header-height))] max-w-screen-lg place-items-center text-center">
+        <div className="-mx-[--page-padding-md] md:mx-0">
+          <div className="pb-8">
+            <Center>
+              <Link href={`/${latestPost.slug}`} variant="shiny" size="small">
+                <AnimatedShinyText className="transition ease-out">
+                  🚀✨ Catch all up changes!
```
Comment (sebald) [inline docs/app/page.tsx:48]: "read latest updates" or "view recent changes"?

File: docs/app/page.tsx [hunk 3]
```diff
@@ -32,191 +34,208 @@ const SectionLink = ({
...
+  const latestPost = getLatestPost();
+  return (
+    <main className="grid justify-center px-[--page-padding-md] xl:px-[--page-padding-xl]">
+      {/* Hero */}
+      <div className="grid h-[calc(100dvh-var(--page-header-height))] max-w-screen-lg place-items-center text-center">
+        <div className="-mx-[--page-padding-md] md:mx-0">
+          <Center>
+            <Link href={`/${latestPost.slug}`} variant="shiny" size="xsmall">
+              <AnimatedShinyText className="transition ease-out">
+                🚀✨ Read last updates!
```
Comment (sebald) [inline docs/app/page.tsx:47]:
```suggestion
                🚀✨ Read lastest updates!
```

Comment (sebald) [reply docs/app/page.tsx:47]: I think this is the "more correct" english?

Comment (sarahgm) [reply docs/app/page.tsx:47]: latest

**Review by sebald — APPROVED** [review]

**Pull request merged**